rpc_jobs/hw_multi_node_aio.yml

```diff
@@ -129,8 +130,26 @@
                 }} //stage
               ) //conditionalStage
               multi_node_aio_prepare.prepare()
+            }}
+
+            deploy_node = "${{INSTANCE_NAME}}-deploy-vm"
+            inventory_content = """
+            [job_nodes:children]
+            hosts
+            [hosts]
+            ${{deploy_node}} ansible_port=2222 ansible_host=${{INSTANCE_IP}}
+            """
+            common.drop_inventory_file(inventory_content)
+            dir("rpc-gating/playbooks"){{
+              stash (
+                name: "pubcloud_inventory",
+                include: "inventory/hosts"
+              )
+            }}
+            ssh_slave.connect(2222)
```

Comment:
If you wrote both nodes to the same inventory, you'd only need to call ssh_slave.connect once? I like that you're connecting both the host and the VM as nodes, provides flexibility.